src/public/scss/gds/template/partials/_header.scss

```diff
@@ -77,12 +76,8 @@
     text-rendering: optimizeLegibility;
     border-bottom: 1px solid transparent;
 
-    background: url('#{$path}gov.uk_logotype_crown.png') no-repeat;
-    background-size: 36px 32px;
-    background-position: 0 0;
```

Comment:
Why do we not need this anymore? If we don't need it, we don't need the few lines below this too that is overwriting the `background-image`.

Reply:
yes thanks agree , removed